monai/transforms/utils.py

```diff
@@ -2509,5 +2511,25 @@ def distance_transform_edt(
     return convert_data_type(r_vals[0] if len(r_vals) == 1 else r_vals, output_type=type(img), device=device)[0]
 
 
+def apply_affine_to_points(data: torch.Tensor, affine: torch.Tensor, dtype: DtypeLike | torch.dtype):
+    """
+    apply affine transformation to a set of points.
+
+    Args:
+        data: input data to apply affine transformation, should be a tensor of shape (C, N, 2 or 3).
```

Comment:
Here also we should mention what C and N represent.

Reply:
Also update this one!

Reply:
I don't see the updated description for C and N, if it's here just marked this as resolved.

Reply:
Sorry for missing this one, updated in https://github.com/Project-MONAI/MONAI/pull/7942/commits/ffab3900ca31b08a0db29036fa5d50e872d1e2bf